### ISSUE TITLE
1.38.2 is not compatible with GitHub actions 

### ISSUE THREAD

**Issue by ben-manes** [semgrep/semgrep#8614]
[1.38.1](https://github.com/returntocorp/semgrep/releases/tag/v1.38.1) fixed and ran, but the subsequent release fails with 
```
Post job cleanup.
/usr/bin/docker exec  496bcb6c2824835048e0090c4a79fb666430ab166351446e8f2e466cfa86f182 sh -c "cat /etc/*release | grep ^ID"
node:internal/fs/utils:347
    throw err;
    ^

Error: EACCES: permission denied, open '/__w/_temp/_runner_file_commands/save_state_22bc749c-1ec4-4a0b-a201-6a53c348a2e5'

**Comment by ben-manes** [comment]
I think you are now trying to [run as root](https://github.com/actions/checkout/issues/1014) which is a security risk for a supply chain attack.

**Comment by ben-manes** [comment]
Closing as fixed in #8613
